docs/src/cli/transfer-tokens.md

```diff
@@ -0,0 +1,98 @@
+# Send and Receive Tokens
+
+## Receive Tokens
+
+To receive tokens, you will need an address for others to send tokens to. In
+Solana, an address is the public key of a *keypair*. There are a variety
+of techniques for generating keypairs. The method you choose will depend on how
+you choose to store keypairs.  Keypairs are stored in *wallets*. Before receiving
+tokens, you'll need to [choose a wallet](choose-a-wallet.md) and
+[generate keys](generate-keys.md). Once completed, you should have a *public key*
+for each keypair you generated. The public key is a long string of base58
+characters. Its length varies from 42 to 43 characters.
```

Comment:
Is this length range correct? 32bytes of 0s is 32char, 32bytes of 255s is 44char.

TBH, maybe too technical a fact for this document?

Reply:
getting it right would be nice. have you seen this written down anywhere?

Reply:
```
    println!("32 0s:    {}", bs58::encode(&vec![0u8; 32]).into_string().len());
    println!("32 255s:  {}", bs58::encode(&vec![255u8; 32]).into_string().len());
```
produces
```
32 0s:    32
32 255s:  44
```